tests/test_tasks_downstream.py

```diff
@@ -201,31 +195,28 @@ def test_install_test(
                 invoke("git-aggregate")
                 invoke("resetdb")
             # Install "mail"
+            assert _install_status("mail") == "uninstalled"
             stdout = invoke("install", "-m", "mail")
-            assert "Executing odoo --stop-after-init --init mail" in stdout
             assert _install_status("mail") == "installed"
             if supported_odoo_version > 8:
                 assert _install_status("utm") == "uninstalled"
-            assert _install_status("note") == "uninstalled"
-            if supported_odoo_version > 8:
                 # Change to "utm" subfolder and install
                 with local.cwd(
                     tmp_path / "odoo" / "custom" / "src" / "odoo" / "addons" / "utm"
                 ):
                     # Install "utm" based on current folder
                     stdout = invoke("install")
-                    assert "Executing odoo --stop-after-init --init utm" in stdout
-                    assert _install_status("mail") == "installed"
-                    assert _install_status("utm") == "installed"
-                    assert _install_status("note") == "uninstalled"
+                assert _install_status("mail") == "installed"
+                assert _install_status("utm") == "installed"
             # Test "note" simple call in init mode (default)
+            assert _install_status("note") == "uninstalled"
             stdout = invoke("test", "-m", "note", "--mode", "init", retcode=None)
             # Ensure "note" was installed and tests ran
             assert _install_status("note") == "installed"
             assert _tests_ran(stdout, supported_odoo_version, "note")
             # Test "note" simple call in update mode
             stdout = invoke("test", "-m", "note", "--mode", "update", retcode=None)
-            assert _tests_ran(stdout, supported_odoo_version, "note", mode="-u")
+            assert _tests_ran(stdout, supported_odoo_version, "note")
```

Comment:
If you are doing `assert`s in the _tests_ran function, this needs to be adapted. Either return true or false in the function, or just do a normal call here.